packages/insomnia-app/app/ui/components/codemirror/code-editor.tsx

```diff
@@ -146,6 +148,10 @@ interface State {
   filter: string;
 }
 
+function isMarkerRange(mark: CodeMirror.Position | CodeMirror.MarkerRange | undefined): mark is CodeMirror.MarkerRange {
+  return Boolean(mark && 'from' in mark);
```

Comment:
can you please refactor this to be simpler and also not use the `in` operator?

```suggestion
  if (!mark) {
    return false;
  }
  return Object.prototype.hasOwnProperty.call(mark, 'from');
```